core/src/main/java/org/jdbi/v3/core/StatementContext.java

```diff
@@ -208,6 +210,16 @@ Cleanables getCleanables() {
         return cleanables;
     }
 
+    public Method getSqlObjectMethod()
```

Comment:
As I think about this more, is there anything wrong with making this property just a string, with a generic name to communicate that it identifies the statement context for tracking purposes (logging, metrics)